.eslintrc.json

```diff
@@ -1,3 +1,29 @@
 {
-    
+    "env": {
+        "browser": true,
+        "es6": true,
+        "jest": true
+      },
+      "parser": "babel-eslint",
+      "parserOptions": {
+        "ecmaFeatures": {
+          "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+      },
+      "extends": ["airbnb", "plugin:react/recommended"],
+      "plugins": ["react"],
+      "rules": {
+        "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+        "react/react-in-jsx-scope": "off",
+        "import/no-unresolved": "off",
+        "no-shadow": "off",
+        "import/prefer-default-export": "off",
```

Comment:
- [ ] Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)